src/Bundle/Command/StubMakeResourceTransformer.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ResourceBundle\Command;
+
+final class StubMakeResourceTransformer extends StubCommand
+{
+    protected static $defaultName = 'make:resource-transformer';
```

Comment:
before that I would even introduce make:resource command :)

Reply:
Yes, one day... 🤣